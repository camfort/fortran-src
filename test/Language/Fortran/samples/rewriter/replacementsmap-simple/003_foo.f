       subroutine aaaa()
       call bbbb1

       call bbbb2
       end

