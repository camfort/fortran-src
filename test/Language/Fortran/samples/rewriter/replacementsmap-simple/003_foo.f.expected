       subroutine aaaa()
       call cccc1

       call cccc2
       end

